### PR TITLE
Add basic documentation writing guide

### LINKED ISSUE #1825: Document content creation and publishing process
### What should be done?

We should document the process of creating content from start to finish for developers and contributors. This should include:

- How to create issues for topic pitch
- How to create a Markdown document based on a provided template and include Docusaurus frontmatter details
- The writing style and tone of our content (also refer to glossary for terminologies).
- Add a section on the review process and responsibilities of the team.

Docusaurus has a way of "hiding" published content in production. They are still accessible if you have the URL. We want to adopt this in our process `(unlisted: true)` for posts that are very much wip, or that need their published dates changed later.

### PR DESCRIPTION
**Pull request by AlekSi**
For everyone to review and agree.

# Description

Closes #1825.

## Readiness checklist

* [ ] I added unit tests for new functionality or bug fixes.
* [ ] I added integration tests for new functionality or bug fixes.
* [ ] I added compatibility tests for new functionality or bug fixes.
* [ ] I made spot refactorings.
* [x] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/docs/contributing/_category_.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Contributing",
+  "position": 5,
+  "link": {
+    "type": "generated-index",
+    "description": "Learn to install FerretDB on your local machine"
```
Comment (Fashander) [inline website/docs/contributing/_category_.json:6]:
```suggestion
    "description": "Follow our guide on how to contribute to FerretDB"
```

File: website/blog/_guide/template.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Files should be in this format `YYYY-MM-DD-shortened-article-name.md` or `YYYY-M
 All images for this blog post - including the banner image - should be stored in a folder with the blogpost date under this directory `(../../static/img/blog)`, for example, `/img/blog/2022-12-29/banner.png`.
 
 Regular blog authors and engineers can be added to `authors.yml`.
-When the author information is present in `authors.yml`, fill the unique author name  in the frontmatter as (`author: name`)
+When the author information is present in `authors.yml`, fill the unique author name in the frontmatter as (`author: name`)
```
Comment (Fashander) [inline website/blog/_guide/template.md:34]:
```suggestion
When the author information is present in `authors.yml`, fill the unique author name in the frontmatter as (`authors: [name tag]`). Ensure to remove all other descriptions in the frontmatter about the author (`author`, `author_url`, `author title`, and `author_image_url`).
```

**Review by Fashander — DISMISSED** [review]
This is really good. Only one comment, and then I'll update the guide with more details in another PR.

**Review by Fashander — APPROVED** [review]

**Review by b1ron — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
:shipit:

**Review by noisersup — APPROVED** [review]
Looks reasonable!

**Review by chilagrow — APPROVED** [review]

**Review by Fashander — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]
👍

**Pull request merged**